Makefile

```diff
@@ -89,12 +89,13 @@ cmd:
 
 lint:
 	which black && exit `black -q --diff --exclude src/north/snmp/src src | wc -l`
+	which black && exit `black -q --diff --exclude "src/north/snmp/src|src/north/gnmi/goldstone/north/gnmi/proto" src | wc -l`
```

Comment:
Do we need to delete the line above?

```
which black && exit `black -q --diff --exclude src/north/snmp/src src | wc -l`
```

Reply:
Yes. It's a mistake. I'll delete the line.

Reply:
done